socorro/schemas/validate_and_test.py

```diff
@@ -32,50 +33,82 @@ def save_processed(self, crash):
         self.combined = crash
 
 
-def main():
+def main(*urls):
     file_path = os.path.join(HERE, 'crash_report.json')
     with open(file_path) as f:
         schema = json.load(f)
     jsonschema.Draft4Validator.check_schema(schema)
     print('{} is a valid JSON schema'.format(file_path))
 
     print('Fetching data...')
-    r = requests.get(
-        API_BASE.format('SuperSearch'),
-        params={
-            'product': 'Firefox',
-            '_columns': ['uuid'],
-            '_facets_size': 0
-        }
-    )
-    search = r.json()
+    uuids = []
+    for url in urls:
+        assert '://' in url, url
+
+        # To make it easy for people, if someone pastes the URL
+        # of a regular SuperSearch page (and not the API URL), then
+        # automatically convert it for them.
+        parsed = urlparse(url)
+        if parsed.path == '/search/':
+            parsed = parsed._replace(path='/api/SuperSearch/')
+            parsed = parsed._replace(fragment=None)
+            url = parsed.geturl()
+        r = requests.get(
+            url,
+            params={
+                '_columns': ['uuid'],
+                '_facets_size': 0
+            }
+        )
+        search = r.json()
+        if not search['total']:
+            print('Warning! {} returned 0 UUIDs'.format(
+                url
+            ))
+        uuids.extend(
+            [x['uuid'] for x in search['hits'] if x['uuid'] not in uuids]
+        )
+    if not urls:
+        r = requests.get(
+            API_BASE.format('SuperSearch'),
+            params={
+                'product': 'Firefox',
+                '_columns': ['uuid'],
+                '_facets_size': 0
+            }
+        )
+        search = r.json()
+        uuids = [x['uuid'] for x in search['hits']]
+
+    assert len(uuids) == len(set(uuids))
```

Comment:
Huh? What's the point of this? Why not just make ``uuids`` a set to begin with? It's going to be very painful if you need to make searches that do not collide.